src/frontends/tensorflow/src/op/extract_image_patches.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op_table.hpp"
+#include "openvino/op/util/attr_types.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "utils.hpp"
+
+using namespace std;
+using namespace ov::opset8;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+
+OutputVector translate_extract_image_patches_op(const NodeContext& node) {
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() >= 0, "ExtractImagePatches must have at least one input.");
+    auto images = node.get_input(0);
+
+    // retrieve attributes for Conv2DBackpropInput
+    auto tf_ksizes = node.get_attribute<std::vector<int64_t>>("ksizes");
+    auto tf_strides = node.get_attribute<std::vector<int64_t>>("strides");
+    auto tf_rates = node.get_attribute<std::vector<int64_t>>("rates");
+    auto tf_padding_type = node.get_attribute<std::string>("padding");
+    ov::op::PadType auto_pad = convert_tf_padding(node, tf_padding_type);
+    TENSORFLOW_OP_VALIDATION(node,
+                             auto_pad == ov::op::PadType::SAME_UPPER || auto_pad == ov::op::PadType::VALID,
+                             "Only SAME_UPPER and VALID padding modes are supported for ExtractImagePatches.");
+
+    // prepare attributes for OpenVINO ConvolutionBackpropData
+    Shape sizes(2), rates(2);
```

Comment:
nit: I would prefer that done in 2 separate lines.

Reply:
Done